ci-operator/step-registry/openshift-qe/node-density-cni/OWNERS

```diff
@@ -0,0 +1,17 @@
+approvers:
```

Comment:
This file should reference the OWNERS file in the main openshift-qe folder like https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/cluster-density-v2/OWNERS
This way we only need to update one file for the whole repo and it'll stay up to date

Reply:
you'll want to delete this file and then create a symbolic link using 
`ln -s ./OWNERS ./<folder_name>`